tests/app.put.test.js

```diff
@@ -0,0 +1,152 @@
+const app = require("../app");
```

Comment:
You should probably rename the file to be better. `updateUserInformation.test.js` would be better.

Reply:
9a4a464832b8a05a2ec4f1893f42397a3c4a522b